util/local_verdaccio_publish

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Always clear storage so the latest versions are published
+# I am not worried about _what_ version number is published
+# Only that it is the latest code
+rm -rf verdaccio/storage/@aws-crypto
+
+# Start verdaccio in the background
+verdaccio -c verdaccio/config.yaml &
+
+# Publish all changed packages the local verdaccio server.
+# Anything that has not been changed will match what is in npm
+npx lerna publish prepatch \
+  --registry http://localhost:4873/ \
+  --yes \
+  --no-changelog \
+  --no-git-tag-version \
+  --no-push \
+  --no-git-reset \
+  --ignore-scripts \
+  --preid ci \
+  --no-verify-access
+
+# The above command will make some modifications,
+# Roll them back
+# Idealy, we would find a way to not have to do this
```

Comment:
Finding a way to not have to do this might require an upstream change to lerna.

Reply:
I agree